make/xml.cmake

```diff
@@ -37,3 +37,99 @@ function(XML_LINT)
 
 endfunction(XML_LINT)
 
+function(XML_SORT)
+  # ~~~
+  # XML_SORT(
+  # NAME
```

Comment:
Should this take a name? Can we auto-create a name from the input file?

Reply:
I recommend have the name be seperate from the file name to avoid target collision.  CMake has GLOBAL target namespace, so collisions can happen.

Reply:
Doesn't depending on the output file name guarantee it'll be unique?